verified_boot_main.md

```diff
@@ -39,25 +39,224 @@
+  - Can only be turned off by a system reset
+
+<!--
+https://eclypsium.com/blog/firmware-security-realizations-part-3-spi-write-protections/
+https://opensecuritytraining.info/IntroBIOS_files/Day2_03_Advanced%20x86%20-%20BIOS%20and%20SMM%20Internals%20-%20SPI%20Flash%20Protection%20Mechanisms.pdf
+https://nixhacker.com/analyse-bios-protection-against-uefi-rootkit/
+https://cdrdv2-public.intel.com/743835/743835-004.pdf, section 27.1.1
+ -->
+
+[^Intel_series_9_pch]: https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/9-series-chipset-pch-datasheet.pdf
+[^Intel_series_9_pch_5-26-2-1] Intel 9 Series Chipset PCH, section 5.26.2.1: https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/9-series-chipset-pch-datasheet.pdf
+[^Intel_series_9_pch_5-26-5-1] Intel 9 Series Chipset PCH, section 5.26.2.1: https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/9-series-chipset-pch-datasheet.pdf
+
 
 ### Intel Boot Guard / AMD Platform Secure Boot
 
+<!-- TODO?? Because the technologies are more or less equivalent, maybe it's
+better to describe what they are about here and then show the differences
+in naming in the two sections -->
+
+The two technologies are mostly equivalent and serve the purpose of:
+- Verifying and Measuring the BIOS firmware
+- Providing a Hardware RTM (HRTM) and RTV (HRTV) for the processes
+  of verified and measured boot
+- Allowing to create uninterrupted Chains of Trust from the hardware up to
+  an operating system
+
+The hardware roots of trust consist of:
+- A secure storage for enrolling keys by the OEM. Possibly using electronic
+fuses[^efuses_wikipedia] that make the keys permamently encoded into the CPU
+- Hardware implementations of basic cryptography operations
+- A small read-only code for verifying the firmware
+
+While similar in essence, Intel Boot Guard and AMD Platform Secure Boot use
+different naming for equivalent steps and elements of the technologies.
+
+#### Intel Boot Guard
+
+Intel Boot Guard[^Intel_introduction_to_key_usage] is a technology that
+implements the RTM and the RTV in hardware.
+The hardware-based RTV works by verifying the initial part of the firmware
+called the `Initial Boot Block (IBB)` using the `Authenticated Code Module (ACM)`
+code embedded in the CPU by the manufacturer. The IBB is verified using the OEM
+keys, that are saved in the CPU registers and can be fused to make them
+permamently read-only. The IBB is the second link of the CTV and continues
+to extend it by verifying the rest of the BIOS firmware.
+
+[^Intel_introduction_to_key_usage]: https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/resources/key-usage-in-integrated-firmware-images.html
+
+#### AMD Platform Secure Boot
+
+The AMD Platform Secure Boot[^AMD_PSB], also called AMD Hardware Validated Boot[^AMD_HVB] on data
+center processors, provides the Hardware RTM and RTV thanks to the
```

Comment:
HVB is the old name of PSB. Nowadays, PSB is the official name and is used everywhere. Quite weird to see HVB used on data center whitepaper,  especially that it is mixed with attestation and virtualized environments. Almost like it is something else. I would simply write:

```

The AMD Platform Secure Boot[^AMD_PSB] (formerly known as AMD Hardware Validated Boot[^AMD_HVB]), provides
```